Problem3/src/problem3/Problem3.java

```diff
@@ -1,16 +1,104 @@
 package problem3;
 
-/**
- *
- * @author cgallinaro
- */
+import java.text.DecimalFormat;
+
 public class Problem3 {
 
-    /**
-     * @param args the command line arguments
-     */
     public static void main(String[] args) {
-        // TODO code application logic here
+        int gradesArray[][]
+                = {
+                    {56, 38, 38, 88, 52, 42},
+                    {85, 35, 91, 94, 60, 47},
+                    {78, 62, 70, 42, 93, 65},
+                    {33, 75, 82, 45, 73, 83},
+                    {60, 80, 71, 82, 73, 38},
+                    {80, 55, 48, 59, 36, 80},
+                    {30, 91, 79, 81, 61, 61},
+                    {47, 54, 68, 64, 73, 65},
+                    {32, 54, 84, 52, 64, 80},
+                    {82, 82, 39, 65, 48, 41},
+                    {65, 81, 81, 38, 93, 86},
+                    {44, 74, 64, 61, 58, 77},
+                    {31, 31, 44, 93, 52, 94}
+                };
+        int studentnumber = 0;
+        int testnumber = 0;
+        System.out.println("Name           Test 1  Test 2  Test 3  Test 4  Test 5  Test 6  Average");
+        System.out.println("-------------  ------  ------  ------  ------  ------  ------  -------");
+        while (studentnumber < 13) {
+            if (studentnumber < 9) {
```

Comment:
You should be using printf to control the width of your output (rather then adjusting the number of spaces with _if_ statements).